src/components/FileMetadata/index.spec.tsx

```diff
@@ -1,34 +1,44 @@
-import * as React from 'react';
 import { shallow } from 'enzyme';
+import * as React from 'react';
```

Comment:
This does not seem like a useful change.

Reply:
I had originally added another import, at which point I realized these were not in alphabetical order. I can revert it though.